pss/pss_test.go

```diff
@@ -81,7 +150,7 @@ func newMockLocalStore(t *testing.T) *localstore.DB {
 		t.Fatal(err)
 	}
 
-	localStore, err := localstore.New(dir, baseKey, &localstore.Options{})
+	localStore, err := localstore.New(dir, baseKey, &localstore.Options{Tags: tags})
```

Comment:
well,  now that you add tags to the localstore, in fact you should not increment on the tag

Reply:
When you mean increment the tag, do you mean accessing it thru
`localstore.tag[0]` ?
Then I could remove the tag also from the Monitor struct and return a channel.
related to this [question](https://github.com/ethersphere/swarm/pull/2159#discussion_r412974166)